src/coreclr/src/jit/error.cpp

```diff
@@ -65,6 +65,16 @@ void DECLSPEC_NORETURN noWay()
     fatal(CORJIT_INTERNALERROR);
 }
 
+/*****************************************************************************/
+void DECLSPEC_NORETURN implLimitation()
+{
+#if MEASURE_FATAL
+    fatal_noWay += 1;
```

Comment:
You should add a new `fatal_implLimitation` variable

Reply:
done.